tigerpath/majors_and_certificates/degrees/BSE_2018.json

```diff
@@ -3,53 +3,50 @@
   "name": "B.S.E.",
   "code": "BSE",
   "year": 2018,
+  "description": "The B.S.E. program at Princeton is intended to educate future leaders in many different areas – including engineering practice and research, business and finance, public service, and other professions – through the teaching of fundamental engineering principles and techniques with their applications to modern problems in a global societal context. To this end, B.S.E. students are challenged to conceptualize and solve technical problems, work together in teams, express themselves clearly and convincingly, evaluate evidence critically, and appreciate the ethical, social, economic, and cultural environments in which they will live and work.",
   "urls": [
     "https://ua.princeton.edu/contents/program-study-degree-bachelor-science-engineering",
     "https://ua.princeton.edu/contents/general-education-requirements",
-    "https://lsi.princeton.edu/integratedscience/freshman-year",
-    "https://writing.princeton.edu/seminars/fall-seminars",
-    "https://kellercenter.princeton.edu/learn/freshman-course-sequence"
+    "https://lsi.princeton.edu/integratedscience/freshman-year"
   ],
   "req_list": [
     {
       "name": "Degree Progress",
       "max_counted": 1,
       "min_needed": "ALL",
-      "description": null,
       "explanation": "B.S.E. students enroll in four courses for the first term of the freshman year and in four or five courses in each succeeding term, following a sequence appropriate to their individual programs. The school requirement for the B.S.E. degree is at least 36 courses in the four years of study. Sophomores, juniors, and seniors must complete at least four courses each term, with a minimum of 17 courses by the start of junior year and 26 courses by the start of senior year.",
       "req_list": [
         {
-          "name": null,
+          "name": "By first semester",
           "max_counted": 1,
-          "min_needed": 1,
-          "description": null,
           "explanation": null,
           "completed_by_semester": 1,
           "num_courses": 4
         },
         {
-          "name": null,
+          "name": "By second semester",
+          "max_counted": 1,
+          "explanation": null,
+          "completed_by_semester": 2,
+          "num_courses": 9
```

Comment:
How'd you get 9 here?

Reply:
I remember seeing somewhere that it said BSE students normally enroll in 4 and 5 courses in their first two semesters.
I don't remember where I saw this, and I can't find it again.

Would you prefer to change it to 8, which is the minimum according to these sources?
https://ua.princeton.edu/contents/academic-standing-leaves-of-absence-and-reinstatement#:~:text=B.S.E.%20Program
https://ua.princeton.edu/contents/program-study-degree-bachelor-science-engineering#:~:text=B.S.E.%20students%20enroll
and
https://ua.princeton.edu/contents/academic-regulations/academic-year#:~:text=B.S.E.%20Program

Reply:
```suggestion
          "num_courses": 8
```